src/components/ContactLinks.js

```diff
@@ -3,13 +3,19 @@ import '../css/ContactLinks.scss';
 import { FontAwesomeIcon} from '@fortawesome/react-fontawesome';
 import { faTwitter, faGithub, faInstagram} from '@fortawesome/free-brands-svg-icons';
 
-function ContactLinks({copyright}) {
+function ContactLinks({copyright, twitter, github, instagram}) {
   return (
   <div className="contact">
     <div className="contactLinks">
-      <FontAwesomeIcon className="contactLinks__icon" icon={faGithub}/>
-      <FontAwesomeIcon className="contactLinks__icon" icon={faTwitter}/>
-      <FontAwesomeIcon className="contactLinks__icon" icon={faInstagram}/>
+     <a href={github} target="_blank" rel="noopener noreferrer" className="contactLinks__link">
```

Comment:
Instead of rendering out multiple `<a href...>`s, take the links array property and `.map` over them returning links with the object content (message me if you want to tackle this together.